node/network/statement-distribution/src/lib.rs

```diff
@@ -239,6 +239,10 @@ struct PeerRelayParentKnowledge {
 	/// connecting again with new peer ids, but we assume that the resulting effective bandwidth
 	/// for such an attack would be too low.
 	large_statement_count: usize,
+
+	/// We have seen a message that come out of view from this peer, so note this fact
+	/// and stop subsequent logging and peer reputation flood.
+	seen_out_of_view: bool,
```

Comment:
This can be an integer so we can count how many times this happened. Does it add any value for debugging ?

Reply:
Yes, I was thinking about adding that as `usize` but I was also not sure if it would be useful.